mkosi.sanitizers/mkosi.postinst

```diff
@@ -9,6 +9,13 @@ if [[ ! -f "$BUILDROOT/$LIBSYSTEMD" ]]; then
     exit 0
 fi
 
+# Sanitizers may make exit() extremely slower. Let's extend the timeout. See issue #35335.
+mkdir -p "$BUILDROOT"/etc/systemd/system.conf.d
+cat >"$BUILDROOT"/etc/systemd/system.conf.d/10-default-timeout.conf <<EOF
+[Manager]
+DefaultTimeoutStopSec=2min
+EOF
```

Comment:
Can you add this to `mkosi.extra` in `mkosi.sanitizers`? We prefer to have declarative stuff over more stuff in scripts